### PR TITLE
Update our fork with upstream changes

### PR DESCRIPTION
**Pull request by Brantone**

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 license           'Apache-2.0'
 description       'Recipes and resources for installing Java and managing certificates'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '4.9000.0'
+version           '4.9001.0'
```
Comment (zirkov) [inline metadata.rb:7]: We'd have to be extra careful about version clashes again

File: Dangerfile [hunk 2]
```diff
@@ -1,14 +1,16 @@
 # Reference: http://danger.systems/reference.html
```
Comment (zirkov) [inline Dangerfile:1]: That's an awesome tool, we have to think about adding it to our projects!

**Review by zirkov — APPROVED** [review]

**Comment by zirkov** [comment]
@Brantone I don't have write access so would leave the merge to you

**Comment by Brantone** [comment]
Added applicable team to "Write"

File: Dangerfile (hunk 2 above)
Comment (Brantone) [reply Dangerfile:1]: Good point, definitely worth adding it elsewhere 👍

**Pull request merged**